lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
I get the same results on the zip that @nick-benoit14 was using.
In light of this, we should guard against this. Never know what we will get.
I'm just not sure which is best.
```ruby
files = zipfile.entries.select(&:file?)
```
or
```ruby
files = zipfile.glob("**/**").select(&:file?)
```

@jamis which do you recommend? glob calls `entries.map` with the pattern so we can get fancy if we need to.

I'm almost inclined to say we use `glob`, but I haven't convinced myself 100%. But I can see just using `entries` as it is less code. So maybe we stick with using entries?